src/caduceus/senderWrapper_test.go

```diff
@@ -46,7 +47,7 @@ func (t *swTransport) RoundTrip(req *http.Request) (*http.Response, error) {
 }
 
 func swGetLogger() log.Logger {
```

Comment:
This function is a meaningless abstraction.  Just replace calls to `swGetLogger` to `logging.DefaultLogger()`